35/documentation.html

```diff
@@ -30,7 +30,7 @@
     <!--#include virtual="toc.html" -->
   </div>
   <div class="right">
-    <!--#include virtual="../includes/_docs_banner.htm" -->
+    <!--//#include virtual="../includes/_docs_banner.htm" -->
```

Comment:
Shouldn't `//` be removed? We want the "_You're viewing documentation for an older version of Kafka - check out our current documentation here._" message to appear, right?

Reply:
If this is the case, it needs to be modified on all the other html doc pages

Reply:
Good catch! Updated.